<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget_video_text" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_play">
         <property name="text">
          <string>play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string>stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_playmode">
         <item>
          <property name="text">
           <string>all</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showoutput">
         <property name="text">
          <string>show output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showinput">
         <property name="text">
          <string>show input</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_default_url">
         <property name="text">
          <string>default url</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_default_url"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_text" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_addcam">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_addcam">
            <property name="text">
             <string>rtsp://192.168.1.95:554/av0_1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_delcam">
            <property name="text">
             <string>del</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_delcam">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_text2" native="true">
         <widget class="QPushButton" name="pushButton_load">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>80</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>load</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>44</y>
            <width>80</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>reboot</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>9</y>
            <width>78</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>save</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>44</y>
            <width>78</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>test</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_netbuffer">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>161</width>
            <height>161</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_clear_buffer">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>80</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>clear buffer</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_netbuffer">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>59</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>net buffer</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_text1" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>connect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox_search"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_search">
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_video" native="true">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_video"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
